cmd/pool/account.go

```diff
@@ -417,6 +418,13 @@ var depositAccountCommand = cli.Command{
 			Usage: "the fee rate expressed in sat/vbyte that " +
 				"should be used for the deposit",
 		},
+		expiryAbsoluteFlag,
+		cli.Uint64Flag{
```

Comment:
nit: extract this flag into a variable as well?

Reply:
I thought about it but one out of the three usages has a default value set and the other two don't... we can have a separated PR consolidating the flags later.